logbook-json/src/test/java/org/zalando/logbook/json/JsonPathBodyFiltersTest.java

```diff
@@ -0,0 +1,87 @@
+package org.zalando.logbook.json;
+
+import com.jayway.jsonpath.Configuration;
+import com.jayway.jsonpath.Option;
+import com.jayway.jsonpath.spi.json.JacksonJsonProvider;
+import com.jayway.jsonpath.spi.json.JsonProvider;
+import com.jayway.jsonpath.spi.mapper.JacksonMappingProvider;
+import com.jayway.jsonpath.spi.mapper.MappingProvider;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.zalando.logbook.BodyFilter;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.zalando.logbook.json.JsonPathBodyFilters.jsonPath;
+
+class JsonPathBodyFiltersTest {
+
+    public static final String CONTENT_TYPE = "application/json";
+
+    @Test
+    public void deleteObjectTest() {
+        BodyFilter filter = jsonPath("$.test.test123").delete();
+        String result = filter.filter(CONTENT_TYPE, "{\"test\": {\"test123\":\"testvalue\"}, \"test2\":\"value\"}");
+        assertThat(result).isEqualTo("{\"test\":{},\"test2\":\"value\"}");
+    }
+
+    @Test
+    public void replaceStringTest() {
+        BodyFilter filter = jsonPath("$.test").replace("XXX");
+        String result = filter.filter(CONTENT_TYPE, "{\"test\": [\"test123\", \"test321\", 1.0], \"test123\": \"testing\"}");
+        assertThat(result).isEqualTo("{\"test\":\"XXX\",\"test123\":\"testing\"}");
+
+        result = filter.filter(CONTENT_TYPE, "{\"test\": 1, \"test123\": \"testing\"}");
```

Comment:
Tests become more readable and manageable if there is, as a rule of thumb, one assertion per test. I'd extract every filter invocation and the corresponding assertion into its own test method.